### ISSUE TITLE
cases breaks autouse with parametrization

### ISSUE THREAD

**Issue by mganisin** [smarie/python-pytest-cases#114]
When

```
$ cat test_this.py
import pytest

@pytest.fixture(autouse=True)
def one():
    pass

@pytest.fixture(params=[1, 2])
def two():
    pass

def test_me(two):
    assert True
```

then

```
pytest --setup-show test_this.py
============================================================================ test session starts =============================================================================
platform linux -- Python 3.6.9, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /tmp/tmp.avOUzRDAMS
collected 2 items

test_this.py
        SETUP    F one
        SETUP    F two[1]
        test_this.py::test_me[1] (fixtures used: one, two).
        TEARDOWN F two[1]
        TEARDOWN F one
        SETUP    F one
        SETUP    F two[2]
        test_this.py::test_me[2] (fixtures used: one, two).
        TEARDOWN F two[2]
        TEARDOWN F one

============================================================================= 2 passed in 0.01s ==============================================================================
```

but

```
pytest --setup-show test_this.py
============================================================================ test session starts =============================================================================
platform linux -- Python 3.6.9, pytest-5.4.3, py-1.9.0, pluggy-0.13.1
rootdir: /tmp/tmp.avOUzRDAMS
plugins: cases-2.0.2
collected 2 items

test_this.py
        SETUP    F two[1]
        test_this.py::test_me[1] (fixtures used: two).
        TEARDOWN F two[1]
        SETUP    F two[2]
        test_this.py::test_me[2] (fixtures used: two).
        TEARDOWN F two[2]

============================================================================= 2 passed in 0.01s ==============================================================================
```

Compare loaded plugins and used fixtures (python version doesn't matter)

**Comment by smarie** [comment]
Indeed, thanks for spotting this ! I'll have a look.

**Comment by smarie** [comment]
fixed in 2.0.3, thanks a lot !
